docs/interest.md

```diff
@@ -25,6 +25,10 @@ $interest = Interest\get_interest();
 // Manually pass data to interest header.
 $data = ['HTTP_INTEREST' =>'Carl Sagan|Richard Feynman'];
 $interest = Interest\get_interest( $data );
+
+// Pass interest to header from query string, ei_interest
+$interest_from_query_string = isset( $_GET['ei_interest'] ) ? $_GET['ei_interest' : '';
```

Comment:
If we're pulling from `$_GET` there should be sanitization, even in the example. We don't want our docs to (inadvertently) promote poor code standards.

```suggestion
$interest_from_query_string = isset( $_GET['ei_interest'] ) ? sanitize_text_field( $_GET['ei_interest'] ) : '';
```